Doc/library/py_compile.rst

```diff
@@ -71,6 +71,8 @@ byte-code cache files in the directory containing the source code.
 
    .. versionchanged:: 3.7
       The *invalidation_mode* parameter was added as specified in :pep:`552`.
+      If the :envvar:`SOURCE_DATE_EPOCH` environment variable is set, this will
+      be forced to CHECKED_HASH.
```

Comment:
This should also be mentioned in the main documentation. In other words, if the "version changed" note went away the docs should still cover what they need to cover. So this should say something like "Detecting the :envvar:`SOURCE_DATE_EPOCH` environment variable to force `CHECKED_HASH` was added."